### ISSUE TITLE
Enabling the interruptions

### ISSUE THREAD

**Issue by Harian-Elyoth** [RobTillaart/GY521#15]
The datasheet explained that we can write into the Interrupt Configuration Register to enable interrupts. And then read into the Interrupt Status Register.
Is that possible to make that from your library ?

**Comment by RobTillaart** [comment]
Have to investigate how to keep it simple.
And find some time ☺️

**Comment by RobTillaart** [comment]
Straightforward functions are quite easy but possibly allow people to "break" the config too.

I can make a separate branch so you can test, ok?

**Comment by RobTillaart** [comment]
@Harian-Elyoth 

Apparently the functions are already in the library - I didn't recall that they were public.

So, you should use the functions, they are already in the library.

```cpp
  uint8_t  setRegister(uint8_t reg, uint8_t value);
  uint8_t  getRegister(uint8_t reg);
```

Details can be found in the PDF  to be found with - Google "mpu-6000 and mpu-6050 register map and descriptions revision 4.2"

**Comment by RobTillaart** [comment]
@Harian-Elyoth 
https://github.com/RobTillaart/GY521/tree/develop branch has a file with register-names.
This makes working with registers a bit easier as you can use names instead of numbers.

Develop might be merged later this week.

**Comment by Harian-Elyoth** [comment]
Thanks ! I'll test that out soon

**Comment by RobTillaart** [comment]
Branch with register names merged and new release made 0.2.3

**Comment by RobTillaart** [comment]
Please reopen if there are still problems.
